### PR TITLE
Team wizz final update

### PR DESCRIPTION
**Pull request by blessedjoe2000**
We completed the task as requested, by adding the prices to the relevant flights and we added the team wizz logo successfully. Please review and tell up if there's anything else we need to update. Fly safe! Team Wizz

### CODE REVIEW

File: .vscode/settings.json [hunk 1]
```diff
@@ -0,0 +1,3 @@
+{
```
Comment (agamm) [inline .vscode/settings.json:1]: Please [.gitignore](https://www.atlassian.com/git/tutorials/saving-changes/gitignore) this file.

File: team_wizz.js [hunk 2]
```diff
@@ -1 +1,33 @@
 // Fetch all routes for the airline and insert into #wizz
+
+(async () => {
```
Comment (agamm) [inline team_wizz.js:3]: Can you convert this into a function?
I'd like to see that it is separated, see next comment on details.

File: team_wizz.js [hunk 3]
```diff
@@ -1 +1,33 @@
...
+  const firstFlight = await fetch("https://flyapi.onrender.com/tlv_ber/wizz");
+  const secondFlight = await fetch("https://flyapi.onrender.com/arn_lhr/wizz");
```
Comment (agamm) [inline team_wizz.js]: Here you have 3 fetches, but what if you need to do 100+ fetches? do you want to hard code the fetches?
I don't think so - be lazy! create a function like `getRoutePrice(...)` so you could do all the work in a generalized manner.

File: team_wizz.js [hunk 4]
```diff
@@ -1 +1,33 @@
...
+  const thirdFlight = await fetch("https://flyapi.onrender.com/lis_nyc/wizz");
+
+  const tlvBer = await firstFlight.json();
+  const arnLhr = await secondFlight.json();
+  const lisNyc = await thirdFlight.json();
+
+  const wizzContainer = document.getElementById("wizz");
```
Comment (agamm) [inline team_wizz.js]: I would also separate the DOM related code into a separate function - so if you want to change it it will be done in one place.

**Review by agamm — CHANGES_REQUESTED** [review]
Code is much cleaner, good job! See comments to proceed.

File: team_wizz.js (hunk 3 above)
Comment (blessedjoe2000) [reply team_wizz.js]: Created getRouteRice function

File: team_wizz.js (hunk 4 above)
Comment (blessedjoe2000) [reply team_wizz.js]: This review wasn't too clear, but I have tried to refactor the code to my own understanding

**Comment by blessedjoe2000** [comment]
Team Wizz updated the review

**Comment by blessedjoe2000** [comment]
Please check if this code can be improved with Promise.all. Give us an insight on improvement if possible. Thanks

**Pull request closed without merging**